src/render_engine/gl3/gl3_render_engine.cpp

```diff
@@ -38,7 +73,7 @@ namespace nova::renderer::rhi {
         glEnableVertexAttribArray(6);
     }
 
-    void Gl4NvRenderEngine::set_num_renderpasses([[maybe_unused]] uint32_t num_renderpasses) {
+    void Gl4NvRenderEngine::set_num_renderpasses(uint32_t /* num_renderpasses */) {
```

Comment:
why the change?

Reply:
_someone_ told me to not use `[[maybe_unused]]` for unused method parameters and instead to comment out their names

Reply:
Will this never use the parameter?

Reply:
The OpenGL backend doesn't. This method lets the Vulkan and D3D12 backends allocate things for their renderpasses. OpenGL has no notion of renderpasses and thus doesn't need to allocate anything for them

Reply:
:+1: